style.css

```diff
@@ -171,53 +195,50 @@ h2 {
 
 /* ----------Media Query---------- */
 
-@media screen and (min-width: 665px) and (max-width: 1120px) {
-  .App {
-    padding: 0 20px;
+@media screen and (min-width: 765px) and (max-width: 1120px) {
+  body {
+    padding: 0 3rem;
   }
-  .header {
-    flex-direction: row;
+
+  .cyf {
+    padding: 0 1rem;
   }
 
-  .App .cyf .about-trainees figure {
+  .cyf .about-trainees figure {
```

Comment:
Do you need this to be so specfied?

Always use the lightest weight you can when writing CSS. Only use what you must. I think many of these rules could be simplified in this way. What do you think?

https://css-tricks.com/strategies-keeping-css-specificity-low/